src/overlays/actors/ovl_kaleido_scope/z_kaleido_equipment.c

```diff
@@ -140,7 +141,7 @@ void KaleidoScope_DrawEquipment(GlobalContext* globalCtx) {
             gDPPipeSync(POLY_OPA_DISP++);
             gSPVertex(POLY_OPA_DISP++, &pauseCtx->equipVtx[j], 4, 0);
 
-            POLY_OPA_DISP = KaleidoScope_QuadTextureIA8(POLY_OPA_DISP, D_02000A00[1], 32, 32, 0);
+            POLY_OPA_DISP = KaleidoScope_QuadTextureIA8(POLY_OPA_DISP, gHUDEmptyMapTex, 32, 32, 0);
```

Comment:
FYI the name `gHUDEmptyMapTex` is incorrect. It's the square around selected items in the inventory and equipment screens. (also not HUD related)

Reply:
Yeah I meant to render that one here